packages/salesforcedx-vscode-lwc/test/vscode-integration/intellisense.test.ts

```diff
@@ -8,217 +8,264 @@
-  let lwcExtension: vscode.Extension<any>;
-  before(async() => {
-    lwcDir = path.join(
-      vscode.workspace.workspaceFolders![0].uri.fsPath,
-      'force-app',
-      'main',
-      'default',
-      'lwc'
-    );
+describe('LWC Intellisense Integration Tests', () => {
 
-    lwcExtension = vscode.extensions.getExtension(
-      'salesforce.salesforcedx-vscode-lwc'
-    ) as vscode.Extension<any>;
-    await lwcExtension.activate();
-  });
+  describe('LWC JS Intellisense Test Suite', function() {
+    // tslint:disable-next-line:no-invalid-this
+    this.timeout(20000);
+    let lwcDir: string;
+    let lwcExtension: Extension<any>;
+    let docUri: Uri;
+    let doc: TextDocument;
+    let editor: TextEditor;
+    let text: string;
+    let startPosition: Position;
+    let endPosition: Position;
+    let selection: Selection;
 
-  afterEach(async () => {
-    try {
-      await vscode.commands.executeCommand(
-        'workbench.action.closeActiveEditor'
+    beforeEach(async() => {
+      lwcDir = path.join(
+        vscode.workspace.workspaceFolders![0].uri.fsPath,
+        'force-app',
+        'main',
+        'default',
+        'lwc'
       );
-    } catch (e) {
-      throw e;
-    }
-  });
+      lwcExtension = extensions.getExtension(
+        'salesforce.salesforcedx-vscode-lwc'
+      ) as Extension<any>;
+      await lwcExtension.activate();
+      docUri = Uri.file(path.join(lwcDir, 'hello', 'hello.js'));
+      doc = await workspace.openTextDocument(docUri);
+      editor = await window.showTextDocument(doc);
+      startPosition = new Position(1, 0);
+      // To provide valid arguments to selection in afterEach, setting default value of endPosition same as startPosition. 
+      endPosition = new Position(1,0);
+    });
 
-  it('lwc extension activation', async function() {
-    expect(lwcExtension.isActive);
-  }).timeout(10000);
-
-  /**
-   * Test that lwc markup intellisense includes standard lwc tags and custom lwc tags
-   */
-  it('LWC Markup Intellisense', async () => {
-    const docUri = vscode.Uri.file(path.join(lwcDir, 'hello', 'hello.html'));
-    const doc = await vscode.workspace.openTextDocument(docUri);
-    const editor = await vscode.window.showTextDocument(doc);
-
-    // We have to have some text or we'll just get generic completions
-    const text = '<c-';
-    const startPosition = new vscode.Position(5, 0);
-    const endPosition = new vscode.Position(
-      startPosition.line,
-      startPosition.character + text.length
-    );
-    const rangeReplace = new vscode.Range(startPosition, endPosition);
-    await editor.edit(editBuilder => {
-      editBuilder.replace(rangeReplace, text);
+    afterEach(() => {
+      try {
+        selection = new Selection(startPosition, endPosition);
+        // We need to clear the editor's line or the input text will change and test will fail.
+        editor.edit(editBuilder => {
+          editBuilder.delete(selection)
+        });
+      } catch (e) {
```

Comment:
Do we have to try/catch here for some reason?  Seems it'd be the same result (a failed afterEach) without it.

Reply:
Yes! It would be the same result: failed afterEach. So have removed it from both instances.